Tests/SwiftDocCUtilitiesTests/Init/InitActionTests.swift

```diff
@@ -0,0 +1,64 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import XCTest
+@testable import SwiftDocCUtilities
+
+final class InitActionTests: XCTestCase {
+    
+    let fileManager = FileManager.default
+    
+    func testInitActionCreatesDocCatalog() throws {
+        let outputURL = try createTemporaryDirectory()
+        var action = try InitAction(
+            catalogOutputDirectory: outputURL,
+            documentationTitle: "MyTestDocumentation",
+            catalogTemplate: .articleOnly
+        )
+        var isDirectory: ObjCBool = false
+        _ = try action.perform(logHandle: .standardOutput)
+        // Test an output folder exists
+        guard fileManager.fileExists(atPath: "\(outputURL.path)/MyTestDocumentation.docc", isDirectory: &isDirectory) else {
+            XCTFail("InitAction failed to create output folder")
+            return
+        }
+        // Test the output folder really is a folder.
+        XCTAssert(isDirectory.boolValue)
+    }
+    
+    func testInitActionCatalogContent() throws {
```

Comment:
What's the intention of this test (compared to the previous `testInitActionCreatesDocCatalog`)? Maybe the title can be more explicit?